src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.iOS.cs

```diff
@@ -1229,7 +1229,12 @@ internal CGPoint GetTargetScrollOffset(NSIndexPath item, ScrollIntoViewAlignment
 
 			nfloat GetBaseOffset()
 			{
-				var frame = LayoutAttributesForItem(item).Frame;
+				var attributes = LayoutAttributesForItem(item);
+				if (attributes == null)
+				{
+					return default;
```

Comment:
@davidjohnoliver does default make sense here ?

Reply:
No, it's not correct for `attributes` to be null here, per discussion above. This bug is already fixed in #1134 (which I'm about to merge).

Reply:
Close PR?